src/test/java/io/jenkins/plugins/oak9/Oak9BuilderTest.java

```diff
@@ -10,17 +10,19 @@
 public class Oak9BuilderTest {
     @Test
     public void testConstructor() {
```

Comment:
The new changes seem fine/in-line with the prior ones, but this test confuses me. It says `testConstructor`, but it's actually testing the getters and setters because all of the setters are called before the values are tested.

Reply:
You're right, this is confusing. I will refactor this into two tests